artifacts/attributes.adoc

```diff
@@ -63,3 +63,6 @@
 :LinkPluginsGuide: https://access.redhat.com/documentation/en-us/red_hat_developer_hub/{product-version}/html-single/confuguring_plugins_in_red_hat_developer_hub/index
 :NameOfPluginsGuide: Configuring plugins in {product}
 
+//:LinkInstallGuide: https://access.redhat.com/documentation/en-us/red_hat_developer_hub/{product-version}/html-single/installation_guide_for_red_hat_developer_hub/index
```

Comment:
```suggestion
:LinkInstallGuide: https://access.redhat.com/documentation/en-us/red_hat_developer_hub/{product-version}/html-single/installation_guide_for_red_hat_developer_hub/index
```

Reply:
And even now, prefer a docs.redhat.com URL.